sideloaded-kit-samples/mp-sideloaded-kit-example/src/ConsoleLoggingKit.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ConsoleLoggingKit.swift
+//  iOSTestApp_Cocoapods_Swift
+//
+//  Created by Ben Baron on 3/1/23.
+//
+
+import Foundation
+import mParticle_Apple_SDK
+
+@objc public class ConsoleLoggingKit: NSObject, MPKitProtocol {
+    // Not used in sideloaded kits
+    public static func kitCode() -> NSNumber { -1 }
+    
+    // Must be true to receive callbacks, can set to true later after any needed initialization is performed
+    public var started: Bool = true
+    
+    public var sideloadedKitCode: NSNumber = 0
```

Comment:
Should this be a NSNumber or an int?

Reply:
That would have been a question for the mparticle-apple-sdk review as it's part of the MPKitProtocol, but I ultimately chose NSNumber to align with the kitCode() method type.